mmcv/cnn/utils/flops_counter.py

```diff
@@ -116,7 +117,9 @@ def get_model_complexity_info(model,
     return flops_count, params_count
 
 
-def flops_to_string(flops, units='GFLOPs', precision=2):
+def flops_to_string(flops: float,
+                    units: Optional[str] = 'GFLOPs',
```

Comment:
```suggestion
                    units: str = 'GFLOPs',
```

Reply:
Please fix it.

Reply:
Inferred from code, the units could be `None`, maybe this is acceptable?